tests/unit/io/Cereal_unittest.cc

```diff
@@ -0,0 +1,174 @@
+#include <shogun/lib/common.h>
+#include <gtest/gtest.h>
+#include "CerealObject.h"
+
+#include <shogun/lib/SGVector.h>
+#include <cereal/archives/json.hpp>
```

Comment:
`save_json` only works for SGObject. Actually the serialization of sgvector and any are not exposed to users. I just did the internal tests of SGVector and Any objects with the naked(?) save/load methods.